### PR TITLE
add EmailMessage.__str__ method

### PR DESCRIPTION
**Pull request by darnel**

### CODE REVIEW

File: pymess/models/emails.py [hunk 1]
```diff
@@ -61,6 +61,9 @@ def friendly_sender(self):
     def failed(self):
         return self.state == self.STATE.ERROR
 
+    def __str__(self):
+        return str(self.recipient)
```
Comment (rubickcz) [inline pymess/models/emails.py]: I would add more information to string representation of the email message. For example:
`recipient@gmail.com: Subject of the message`.

Comment (darnel) [reply pymess/models/emails.py]: This way it's consistent with [sms model](https://github.com/druids/django-pymess/blob/master/pymess/models/sms.py#L69)

Comment (rubickcz) [reply pymess/models/emails.py]: Well.... ok, but just recipient is IMO not descriptive enought for the whole object.

Comment (darnel) [reply pymess/models/emails.py]: Agree, maybe both models should be fixed. Let's master druid resolves it.

**Pull request merged**